### ISSUE TITLE
Can't log into a Microsoft account on AMD64 SteamOS

### ISSUE THREAD

**Issue by tomrow** [minecraft-linux/mcpelauncher-manifest#910]
**Describe the bug**
I can't log in to a Microsoft Account in-game on my Steam Deck.

**To Reproduce**
Select the Sign In button on the title screen or in Settings in-game. This will result in a "Drowned" error.
Checking with the launcher's troubleshooter notes that the MSA daemon was not found.

**Expected behavior**
I expected to be presented with an XBOX Live login prompt or a QR code.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: SteamOS 3.5.3
 - Version 0.12.0 Build 72
 - Installation Type: Flatpak from Flathub

**Additional context**
This works correctly on ARM64 Raspberry Pi OS Bullseye.

**Comment by ChristopherHX** [comment]
> OS: SteamOS 3.5.3

A recent SteamOS broke xbox login. Reinstalling an previous version of Steam OS can fix the issue.

You are running an OS that I never had used, so I have not even a clear picture what is wrong with the launcher on your OS.

This works correctly on
- ubuntu 22.04 x86_64
- ubuntu 18.04 arm64

At this point the issue has not expanded yet to distros I'm willing to debug.

**Comment by jerome-h-dev** [comment]
Hi @ChristopherHX, im aware you guys dont currently support `SteamOS` (at the moment 🙏 ) 

I have the same issue as @tomrow 

Question I have is given this launcher previously worked on the SteamDeck, can you think of any reason why the launcher would get the same result inside a valid OS inside a VM?

**TLDR**

I explored a work around an wanted your opinion on the results if you dont mind.

On the steam desktop I installed `ubuntu 22.04 x86_64` inside VM.

Then from the VM I used Flatpak to install the launcher as per the wiki guide.

Launched game fine, but had the exact same response from the `Sign In` button. "Drowned"

Ive ran out of ideas on a workaround at this point and dont have capacity/knowledge to fix it on a folk.

Thanks in advance

**Comment by ChristopherHX** [comment]
>  can you think of any reason why the launcher would get the same result inside a valid OS inside a VM?

Network issues might be able to be inherited by VM's...

Does this happen on another VM Host OS? For example on windows as host OS I had no such problem.

Does it happen without a sandbox (AppImage, deb, rpm), flatpak sandbox can make network issues even worse.

![image](https://github.com/minecraft-linux/mcpelauncher-manifest/assets/44845461/742862ea-67cc-4ab9-a5e8-ee24bf2542b8)

As of Today is everything fine in a ubuntu 22.04 VM installed from a fresh iso into a hyper-v instance.

**Note:** installed by appimage, after doing `sudo apt install libfuse2`


Errors until succesful sign in
- First sign in popup sign in did more or less nothing
- Sign in from home screen
  - opened the webview and allowed to login
  - the second time the webview open it auto closed
  - Error Code "Creeper", who on earth had that error name idea. Useless message.
 - Second Sign in from home screen
   - passed without issues
  
  No I have no idea how that sign in code ever worked, it's an game mod based on decompiled java source code.

> exact same response from the `Sign In` button

There are sign in buttons on multiple places..
Usually only the one on the home screen works, after the game had enough time to do some background work..

The sign in popup with a similar sign in button has a success rate of ca. 0%.

**Comment by ChristopherHX** [comment]
Errors of flatpak in the VM until sign in
- "sign in now" error for < 1s Not shure what it said closed itself
- sign in
  - webview opend
  - after signing in it hang up with the signing in message...,
  - closed the game, due to beeing stuck
- sign in from home screen
  - black screen instead of webview
- sign in from home screen
  - sign in window opened allowed sign in
  - black screen instead of webview 
- sign in from home screen
  - sign in window opened allowed sign in
  - webview to confirm sign in opened
  - sign in succeeded

**Comment by ChristopherHX** [comment]
Timezone Dublin cannot login

Try changing your timezone to Europe/London to fix the problem

ref https://github.com/minecraft-linux/mcpelauncher-manifest/issues/945

**Comment by Killerminigun** [comment]
2025 and this is STILL NOT FIXED
